src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Planta.kt

```diff
@@ -1,39 +1,31 @@
 package ar.edu.unahur.obj2.semillasAlViento
 
-abstract class Planta(val anioObtencionSemilla: Int, var altura: Float) {
-  fun esFuerte() = this.horasDeSolQueTolera() > 10
-
-  fun parcelaTieneComplicaciones(parcela: Parcela) =
-    parcela.plantas.any { it.horasDeSolQueTolera() < parcela.horasSolPorDia }
-
-  abstract fun horasDeSolQueTolera(): Int
-  abstract fun daSemillas(): Boolean
+abstract class Planta(val anioObtencionSemilla: Int, val altura: Float) { //SIMPLICIDAD val altura- La altura NUNCA cambiara.
+    fun esFuerte() = this.horasDeSolQueTolera() > 10
+    abstract fun condicionAlternativa(): Boolean // para evaluar por un lado si esFuerte() y por otro la condicionAlternativa()
+    abstract fun horasDeSolQueTolera(): Int
+    abstract fun daSemillas(): Boolean
```

Comment:
Hay dos subclases que implementan esto como `this.esFuerte() || this.condicionAlternativa()`, así que esta podría ser la implementación default